Source/ProceduralFairings/FairingSide.cs

```diff
@@ -842,5 +843,19 @@ public void rebuildMesh ()
 
             m.triangles = tri;
         }
+
+        public IEnumerator SetOffset(Vector3 offset, float time = 0.3f)
+        {
+            var mf = part.FindModelComponent<MeshFilter>("model");
+            var lp = mf.transform.localPosition;
+            float elapsedTime = 0f;
+
+            while (elapsedTime < time)
+            {
+                mf.transform.localPosition = Vector3.Lerp(lp, offset, (elapsedTime / time));
+                elapsedTime += Time.deltaTime;
+                yield return null;
+            }
```

Comment:
This will stop just short of offset, because the last loop execution is for elapsedTime < time.
Upon exiting the while, add one last line:  `mf.transform.localPosition = offset`